tests/utils/mock_butler.py

```diff
@@ -181,11 +258,16 @@ def getDataset(self, datid):
     def get(self, ref, collections=None, dataId=None):
         orig_ref = ref
 
+        # this covers tests in region_search because they pass dataId (that are
+        # actuall dataRefs) with a component name as ref. TODO: fix this?
+        if dataId is not None and isinstance(ref, str):
+            orig_ref = dataId.makeComponentRef(ref.split(".")[-1])
```

Comment:
Yeah I don't remember how this happened based on my early butler testing, but I should fix this. Thanks!